Cut v0.2 branch + add v0.2.x tag (#820)
knative/docs: doc-releases.md

# Documentation Releases

> Use the following list to view the version of the Knative documentation for your

Reviewer: ```suggestion
The following list shows the available versions of Knative documentation.
```

> version of Knative cluster that you installed.

Reviewer: ```suggestion
Select the version that matches your installed version of Knative.
```
